src/ptf/testutils.py

```diff
@@ -2394,6 +2395,34 @@ def test_param_get(key, default=None):
     except:
         return default
 
+def test_port_params_get(default={}):
+    """
+    Return all the port info values passed via --interface if present
+
+    @param default Default dictionary to use if no params were provided or if
+    the provided string could not be "exec'd".
+
+    """
+    if PORT_INFO is None:
+        return default
+    return PORT_INFO
+
+def port_param_get(port, key, default=None):
```

Comment:
shouldn't the first function be called `port_params_get`? I don't understand the `test_` prefix.

Reply:
The redundant prefix have been removed